src/main/java/frc/robot/commands/GoHomeSequence.java

```diff
@@ -8,36 +8,34 @@
 import frc.robot.subsystems.elevator.Elevator.ElevatorPosition;
 import frc.robot.subsystems.intake.Intake;
 import frc.robot.subsystems.intake.Intake.IntakePosition;
+import frc.robot.subsystems.shooter.Shooter;
+import frc.robot.subsystems.shooter.Shooter.ShooterMode;
 
 /**
- * brings the elevator and intake to their respective home positions in a "smart" way so they do not
- * end up attacking each other accidentally
+ * Brings the elevator, intake, an shooter to their respective home positions in a "smart" way so
```

Comment:
`an shooter` -> `and shooter` (i wouldn't request changes just for this but if you're already changing other stuff, might as well)

Reply:
Sure fixed